test/functional/bip65-cltv-p2p.py

```diff
@@ -33,19 +26,34 @@ def cltv_invalidate(tx):
     tx.vin[0].scriptSig = CScript([OP_1NEGATE, OP_CHECKLOCKTIMEVERIFY, OP_DROP] +
                                   list(CScript(tx.vin[0].scriptSig)))
 
+def cltv_validate(node, tx, height):
+    '''Modify the signature in vin 0 of the tx to pass CLTV
+    Prepends <height> CLTV DROP in the scriptSig, and sets
+    the locktime to height'''
+    tx.vin[0].nSequence = 0
+    tx.nLockTime = height
+
+    # Need to re-sign, since nSequence and nLockTime changed
+    signed_result = node.signrawtransaction(ToHex(tx))
+    new_tx = CTransaction()
+    f = BytesIO(hex_str_to_bytes(signed_result['hex']))
+    new_tx.deserialize(f)
```

Comment:
nit: perhaps combine this with the line above, rather than have a throwaway variable.